search/scorer/scorer_knn.go

```diff
@@ -71,7 +71,7 @@ func NewKNNQueryScorer(queryVector []float32, queryField string, queryBoost floa
 
 // Score used when the knnMatch.Score = 0 ->
 // the query and indexed vector are exactly the same.
-const maxKNNScore = math.MaxFloat64
+const maxKNNScore = math.MaxFloat32
```

Comment:
There a reason you changed this?

Reply:
with grpc i encountered a marshalling error where returning math.MaxFloat64, it got resolved however by setting it to MaxFloat32. Looks like grpc doesnt work too well with marshalling and unmarshalling a MaxFloat64 value

Reply:
Ack.